compiler/rustc_session/src/config.rs

```diff
@@ -1718,6 +1738,10 @@ pub fn build_session_options(matches: &getopts::Matches) -> Options {
     let mut debugging_opts = build_debugging_options(matches, error_format);
     check_debug_option_stability(&debugging_opts, error_format, json_rendered);
 
+    if !debugging_opts.unstable_options && json_unused_externs {
+        early_error(error_format, "--json=unused-externs is unstable: -Z unstable-options");
```

Comment:
Can we use the standard message here? https://github.com/rust-lang/rust/blob/8e8939b8045b7af6076fb718e2e298844aaf4650/compiler/rustc_session/src/config.rs#L1980-L1987

Fine to just duplicate it, though.

Reply:
I think this still isn't implemented.